SCRWebService/README.md

```diff
@@ -0,0 +1,10 @@
+####Summary Care Record Web Service
+The goal of this section is to produce a web service which provides a simple interface with which 
```

Comment:
V. minor/optional: It feels a bit naff to be saying _The goal of this section_, when you could instead say something like _This is a web service..._